packages/metals-vscode/src/testExplorer/analyzeTestRun.ts

```diff
@@ -162,5 +163,21 @@ function extractErrorMessages(failed: Failed[]): vscode.TestMessage[] {
 }
 
 function toTestMessage(error: string): vscode.TestMessage {
-  return { message: ansicolor.strip(error) };
+  const message = ansicolor.strip(error);
+  const regex = /[\\\/]([^\s]*):(\d+)/;
+  const match = regex.exec(message);
+
+  if (match) {
+    const [, file, line] = match;
+    const filePath = vscode.Uri.file(file).fsPath;
```

Comment:
`file` will just match the file name, not the full path, right? So `vscode.Uri.file(file).fsPath` will create a full path out of it?

Reply:
Uri.file creates an actual uri out if a path, fsPath will print a proper file system path with windows issues handled. We might be able to just use file, but it feels like this is safer